design/comments.go

```diff
@@ -44,6 +47,9 @@ var createCommentAttributes = a.Type("CreateCommentAttributes", func() {
 		a.MinLength(1) // Empty comment not allowed
 		a.Example("This is really interesting")
 	})
+	a.Attribute("markup", d.String, "The comment markup associated with the body", func() {
```

Comment:
it is outsourced twice just as `body` is, because it is used in the types associated with the create and update operations